src/app/pages/checkout-payment/checkout-payment/checkout-payment.component.ts

```diff
@@ -155,9 +158,17 @@ export class CheckoutPaymentComponent implements OnInit, OnChanges, OnDestroy {
    * opens the payment parameter form for a payment method to create a new payment instrument
    */
   openPaymentParameterForm(index: number) {
+    // unselecting the payment instrument
+    if (this.basket.payment && this.basket.payment.paymentInstrument.id) {
+      (document.getElementById(
```

Comment:
This code cannot be used - it is not allowed to reference elements this way. Which problem do you want to solve?

Reply:
We are trying to deselect the radio button and payment instrument on click of 'add new instrument'.